advanced/Scripts/piholeDebug.sh

```diff
@@ -951,8 +952,8 @@ list_files_in_dir() {
             # If it's Web server error log, just give the first 25 lines
             "${PIHOLE_WEB_SERVER_ERROR_LOG_FILE}") make_array_from_file "${dir_to_parse}/${each_file}" 25
```

Comment:
@jacobsalmela just realized that we don't need the `head` but only the `tail` from the `${PIHOLE_WEB_SERVER_ERROR_LOG_FILE}`.